### PR TITLE
[FIX] website_sale:Allow Quantity label on cart page to be editable via website editor

### PR DESCRIPTION
**Pull request by bvanjeelharia**
Description of the issue/feature this PR addresses:
Fixes https://github.com/odoo/odoo/issues/135163
Current behavior before PR:
Quantity label cannot be changed on cart page via website editor
Desired behavior after PR is merged:
Quantity label can changed on cart page via website editor

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by AntoineVDV** [comment]
@odoo/rd-website Could you have your input here? It works, but I have no expertise in the website editor, so I can't tell if it's the proper way to do this.

File: addons/website_sale/views/templates.xml
Comment (rdeodoo) [inline addons/website_sale/views/templates.xml:1397]:
```xml
                        <span t-if="show_qty">
                            Quantity
                        </span>
```
Note that this would work too regarding edition capability, don't change it tho, it's less stable friendly as it changes XML structure (adding a new one is fine-ish). Just mentioning this as if we think about it in master forward port, it's worth changing.

**Review by rdeodoo — APPROVED** [review]
This is LGTM.
I am not sure it's totally worth, it could break xpath targetting `text()` to change this wording, but it's surely overthinking.
Other than that, (very) quickly grepping `<t .*>\n\s+\w` against `website_*` doesn't show a lot of such occurences. I am not sure we could enforce or do something about it.

**Comment by AntoineVDV** [comment]
@rdeodoo Remind about this PR, as the author doesn't have r+ rights

**Comment by AntoineVDV** [comment]
@rdeodoo From my point of view, it made sense, but since you raised concerns I preferred to wait for your definitive approval rather than overrule it :p

**Pull request closed without merging**
